src/platforms/apple/config.yml

```diff
@@ -4,3 +4,7 @@ caseStyle: canonical
 supportLevel: production
 aliases:
   - cocoa
+categories:
+  - mobile
+  - desktop
+  - server
```

Comment:
Wouldn't really work on server due to single hub

Reply:
Whoops you are both right @marandaneto and @bruno-garcia. It should work on server but it doesn't really.